lib/util/src/hse_err.c

```diff
@@ -13,11 +13,10 @@
 
 #include <bsd/string.h>
 
-char hse_merr_bug0[] _merr_attributes = "hse_merr_bug0u";
-char hse_merr_bug1[] _merr_attributes = "hse_merr_bug1u";
-char hse_merr_bug2[] _merr_attributes = "hse_merr_bug2u";
-char hse_merr_bug3[] _merr_attributes = "hse_merr_bug3u";
-char hse_merr_base[] _merr_attributes = "hse_merr_baseu";
+char invalid_file_bug0[] _merr_attributes = "invalid_file_bug0";
```

Comment:
I would say prefixing these with merr still has value. Being able to grep the lib for merr has been useful in the past.